metadata.json

```
@@ -35,6 +35,10 @@
     {
       "name": "puppet/extlib",
       "version_requirement": ">= 3.0.0 < 5.0.0"
+    },
+    {
+      "name": "puppet/redis",
+      "version_requirement": ">= 4.2.0 < 5.0.0"
```

FYI: I just released 5.0.0 which includes the support we want.
```suggestion
      "version_requirement": ">= 5.0.0 < 6.0.0"
```

I'm about to test this on a real system with this patch.

---

Awesome! :heart:  Big :+1: